src/pages/TagsField.tsx

```diff
@@ -22,23 +22,28 @@ interface TagsFieldProps {
 
 const TagsField: FC<TagsFieldProps> = ({ setAllChangesSaved }) => {
   const { t } = useTranslation();
-  const { values, setFieldValue } = useFormikContext<ResourceWrapper>();
+  const { values, setFieldValue, resetForm } = useFormikContext<ResourceWrapper>();
   const [saveTagStatus, setSaveTagsStatus] = useState(StatusCode.ACCEPTED);
 
   const saveTagsChanging = async (name: string, value: string[]) => {
     setAllChangesSaved(false);
+
     try {
-      const stringArrayValue = value as string[];
-      const newTags = stringArrayValue.filter((tag) => !values.resource.tags?.includes(tag));
+      const promiseArray: Promise<any>[] = [];
```

Comment:
denne trengs ikke så lenge vi kjører async await. Kan ta await på hvert kall isteden

Reply:
Men jeg bryr meg ikke om resultatet. Jeg bryr meg kun om når alle sammen er ferdig. I tillegg så får jeg ikke lov til å ha await inni array.map( () => forbudt med await her ). 
Og da vil jo kallene kjøre én og én i stedet for i parallell, som da gjør at saveTagsChange() kjører tregere.

Reply:
ah. i så fall - kjør på :-)